packages/ui/src/test/App.test.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { MemoryRouter } from 'react-router-dom';
-import App from './App';
+import App from '../App';
```

Comment:
actually i would prefer put test files in the same directory, it'll be easier to manage when your project structure becomes more complex. we can discuss on this.